packages/lesswrong/server/migrations/2022-01-12-updateQuadraticVotes-2.ts

```diff
@@ -0,0 +1,107 @@
+import { registerMigration } from './migrationUtils';
+import ReviewVotes from '../../lib/collections/reviewVotes/collection';
+import { REVIEW_YEAR } from '../../lib/reviewUtils';
+import groupBy from 'lodash/groupBy';
+import { Posts } from '../../lib/collections/posts';
+import Users from '../../lib/collections/users/collection';
+
+
+const voteMap = {
+  1: { 
+    cost: 45,
+    value: -9 
+  },
+  2: { 
+    cost: 10,
+    value: -4 
+  },
+  3: { 
+    cost: 1,
+    value: -1 
+  },
+  4: { 
+    cost: 0,
+    value: 0 
+  },
+  5: { 
+    cost: 1,
+    value: 1 
+  },
+  6: { 
+    cost: 10,
+    value: 4 
+  },
+  7: { 
+    cost: 45,
+    value: 9 
+  },
+}
+
+const getCost = (vote) => voteMap[vote.qualitativeScore].cost
+const getValue = (vote) => voteMap[vote.qualitativeScore].value
+
+// TODO: Write a better version of this migration which properly normalizes vote strength
+registerMigration({
```

Comment:
I assume this is entirely copy-pasta-ed without change and I don't need to review it?

Reply:
(Copy pasta is fine here, IMO, as migrations are one and done in this codebase.)